.changeset/four-colts-push.md

```diff
@@ -0,0 +1,7 @@
+---
+"@bi
+"@bigtest/webdriver": minor
+"@bigtest/server": minor
```

Comment:
webdriver and server should get a patch release automatically because they depend directly on atom. Then hitting patch on `bigtest` so that we can pull it in with a single bump.

```suggestion
"@bigtest/atom": minor
"bigtest": patch
```